packages/routers/src/__tests__/CommonActions.test.tsx

```diff
@@ -3,6 +3,6 @@ import * as CommonActions from '../CommonActions';
 it('throws if NAVIGATE is called without key or name', () => {
```

Comment:
```suggestion
it('throws if NAVIGATE is called without name', () => {
```

To avoid accumulating legacy debt we should probably change it here as well